lib/udSDK_example.dart

```diff
@@ -0,0 +1,131 @@
+import 'dart:developer';
+import 'dart:ffi';
+import 'dart:io';
+import 'dart:typed_data';
+
+// import 'package:path_provider/path_provider.dart';
+
+import 'package:vaultSDK/udContext.dart';
+import 'package:vaultSDK/udRenderContext.dart';
+import 'package:vaultSDK/udRenderTarget.dart';
+import 'package:vaultSDK/udPointCloud.dart';
+import 'package:vaultSDK/udConfig.dart';
+
+const List<double> cameraMatrix = [
+  1, 0, 0, 0, //
+  0, 1, 0, 0, //
+  0, 0, 1, 0, //
+  5, -75, 5, 1 //
+];
+
+Future<File> writeBitmap(List<int> bytes) async {
+  // final directory = await getExternalStorageDirectory();
+  // final path = directory.path;
+  const path = "/storage/emulated/0/Android/data/com.example.Pointfluent/files";
```

Comment:
I used [this package](https://pub.dev/packages/path_provider) when I was doing some initial testing of Flutter, I'm not sure if it's any good but it gave me what I wanted.

Reply:
Did try to use this but it doesn't work in this context as it throws an exception that Flutter needs to be initialized before using. Here we are just using dart without Flutter. Good to know for integrating to main app though 👍

Reply:
Oh wow, it says Flutter all over that link, how did I not see that, oops!